lib/ReactViews/Map/Panels/HelpPanel/HelpVideoPanel.jsx

```diff
@@ -49,15 +50,15 @@ class HelpVideoPanel extends React.Component {
       [Styles.shiftedToRight]:
         !isExpanded ||
         !this.props.viewState.showHelpMenu ||
-        this.props.viewState.topElement !== "HelpPanel",
-      [Styles.isHidden]: !itemSelected // when the item isn't selected
+        this.props.viewState.topElement !== "HelpPanel"
     });
     return (
-      <div className={className}>
+      itemSelected && <div className={className}>
```

Comment:
Render this component only when it is selected. Not really sure why it was implemented this way. In my testing with videos that have autoplay enabled, this resulted in the user clicking one video but another video gets played.